docs/en/latest/how-to-build.md

```diff
@@ -21,36 +21,37 @@ title: How to build Apache APISIX
 #
 -->
 
-## Step 1: Install dependencies
-
-The Apache APISIX runtime environment requires dependencies on NGINX and etcd.
-
-Before installing Apache APISIX, please install dependencies according to the operating system you are using. We provide the dependencies installation instructions for **CentOS7**, **Fedora 31 & 32**, **Ubuntu 16.04 & 18.04**, **Debian 9 & 10**, and **MacOS**, please refer to [Install Dependencies](install-dependencies.md) for more details.
-
-## Step 2: Install Apache APISIX
+## Step 1: Install Apache APISIX
 
 You can install Apache APISIX via RPM Repository, RPM package, Docker, Helm Chart, and source release package. Please choose one from the following options.
```

Comment:
```suggestion
You can install Apache APISIX via RPM Repository, Docker, Helm Chart, and source release package. Please choose one from the following options.
```

Reply:
Hi, @spacewander . I suggest Keep the `RPM package` description, some companies need install APISIX locally，so they will install APISIX via RPM package.

Reply:
hi @zaunist , RPM Repository and RPM package are same meanings.

Reply:
> hi @zaunist , RPM Repository and RPM package are same meanings.

Got it.